src/federation/inbox.ts

```diff
@@ -222,6 +235,18 @@ export async function onFollowRejected(
     inboxLogger.debug("Invalid actor: {actor}", { actor });
     return;
   }
+
+  const isRelayAccept = reject.objectId?.hash.includes("#relay-follows/");
```

Comment:
Wouldn't it be more appropriate to use `startsWith()` instead of `includes()` here?

Reply:
Also, shouldn't it be `isRelayReject`?